### PR TITLE
Fix error message on mobile image upload, close #5396

### LINKED ISSUE #5396: Error message broken for file upload (>4mb) on mobile interface
I just got notice that adding photos with size > allowed size is silently failing (or with error message without body) on mobile version.

I reproduced this with mobile Firefox for Android, see screenshot below.
screenshot_2014-11-15-11-47-20

Desired behaviour: Show an error message telling the user that the file could not be uploaded because it was too big (showing the limit, so the user can resize the file to fit).

(EDIT: Whoaaa... big picture... sorry...)

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/6067364-error-message-broken-for-file-upload-4mb-on-mobile-interface?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Flaburgan**

### CODE REVIEW

File: app/assets/javascripts/mobile/mobile_file_uploader.js [hunk 1]
```diff
@@ -2,15 +2,14 @@
...
-
    var aspectIds = gon.preloads.aspect_ids;
 
    new qq.FileUploaderBasic({
        element: document.getElementById('file-upload-publisher'),
        params: {'photo' : {'pending' : 'true', 'aspect_ids' : aspectIds},},
        allowedExtensions: ['jpg', 'jpeg', 'png', 'gif', 'tiff'],
        action: "/photos",
-       debug: true,
+       debug: false,
```
Comment (Flaburgan) [inline app/assets/javascripts/mobile/mobile_file_uploader.js]: Not needed I guess, according to https://github.com/Valums-File-Uploader/file-uploader/blob/master/client/fileuploader.js#L270

It works with debug set to false on Firefox and Chromium at least.

**Comment by denschub** [comment]
Merged.

**Comment by Flaburgan** [comment]
Thank you ;)

**Comment by denschub** [comment]
You're welcome!

**Pull request merged**